### PR TITLE
Airport service unit testing

### PR DESCRIPTION
**Pull request by ronhenson**

### CODE REVIEW

File: airport-service/pom.xml [hunk 1]
```diff
@@ -44,29 +44,46 @@
...
-				<groupId>org.springframework.boot</groupId>
-				<artifactId>spring-boot-maven-plugin</artifactId>
-			</plugin>
+				<plugin>
+						<groupId>org.springframework.boot</groupId>
+						<artifactId>spring-boot-maven-plugin</artifactId>
+						<configuration>
+								<excludes>
+										<exclude>
+												<groupId>org.projectlombok</groupId>
```
Comment (aaronpxavier) [inline airport-service/pom.xml]: fix formatting please.

**Review by aaronpxavier — DISMISSED** [review]

File: airport-service/src/main/java/com/ss/UtopiaAirlines/entity/Airport.java [hunk 2]
```diff
@@ -35,7 +39,7 @@ public String getName() {
 	public void setName(String name) {
 		this.name = name;
 	}
-	
-	
-	
+
+
+
 }
```
Comment (dereklance) [inline airport-service/src/main/java/com/ss/UtopiaAirlines/entity/Airport.java]: newline

File: airport-service/src/test/java/com/ss/UtopiaAirlines/service/AirportServiceTests.java [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+public class AirportServiceTests {
+
+  @Mock
+  private AirportDao airportDao;
+
+  @InjectMocks
+  private AirportService airportService;
+
+  private final String NOT_FOUND = "000";
+  private final String IATALIDENT = "PDX";
```
Comment (dereklance) [inline airport-service/src/test/java/com/ss/UtopiaAirlines/service/AirportServiceTests.java]: fix spelling: `IATA_IDENT`

File: airport-service/src/test/java/com/ss/UtopiaAirlines/service/AirportServiceTests.java [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+ @DisplayName("update airport, expect True")
+ void test4() throws ResourceDoesNotExistsException {
+     assertThat(AIRPORT, samePropertyValuesAs(airportService.updateAirport(AIRPORT)));
+ }
+
+ @Test
+ @DisplayName("delete airport by Id, expect exception ")
+ void test5() {
+   assertThrows(ResourceDoesNotExistsException.class, () -> {
+     airportService.deleteAirport(IATALIDENTDEL);
```
Comment (dereklance) [inline airport-service/src/test/java/com/ss/UtopiaAirlines/service/AirportServiceTests.java]: should use `NOT_FOUND` for this test, you are not mocking `IATALIDENTDEL`. we can remove `IATALIDENTDEL` from the class.

**Review by dereklance — CHANGES_REQUESTED** [review]

**Review by dereklance — APPROVED** [review]

**Pull request merged**